src/AppBar/AppBar.js

```diff
@@ -39,12 +39,10 @@ export default class AppBar extends Component {
      * The CSS class name of the root element.
      */
     className: PropTypes.string,
-    primary: PropTypes.bool,
   };
 
   static defaultProps = {
     accent: false,
-    primary: true,
```

Comment:
`primary` and `accent` as mutually exclusive. I was planning refactor that into a `variation: ['primary', 'accent']`  type. That would affect quite some components.